keras_cv/layers/regularization/squeeze_excite_test.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import tensorflow as tf
+
+from keras_cv.layers import SqueezeAndExciteBlock2D
+
+
+class SqueezeAndExciteBlock2DTest(tf.test.TestCase):
```

Comment:
please add a test case covering custom activations: https://github.com/keras-team/keras/issues/4076#issue-183230193

Reply:
@LukeWood 
Added test. 
Please see #544 before this one. I have changed the docstring assuming that one is merged before this one.